### DISCUSSION TITLE
Stars are counted twice

### DISCUSSION THREAD

**Discussion by OussamaRomdhane** [community/community#58375] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

I have 6 starred repositories but the star count shows "10", while adding some stars to lists, I noticed they were counted twice

![Screenshot from 2023-06-18 17-30-42](https://github.com/community/community/assets/4337769/b549effc-3cb6-4aef-b9e0-b624f374bfab)

**Comment by jshtuazon2** [comment]
58375

**Comment by jayjayson** [comment]
Don't worry about it

**Comment by hankqqu** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> Hello,
> 
> 
> 
> I have 6 starred repositories but the star count shows "10", while adding some stars to lists, I noticed they were counted twice
> 
> 
> 
> ![Screenshot from 2023-06-18 17-30-42](https://github.com/community/community/assets/4337769/b549effc-3cb6-4aef-b9e0-b624f374bfab)
> 
>

**Comment by hankqqu** [comment]
https://github.com/alist-org/alist/issues/4617
